### PR TITLE
ceph-ansible-prs: Don't run auto job if only docs were modified

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml [hunk 1]
```diff
@@ -233,10 +233,23 @@
...
-          !include-raw-escape:
-            - ../../../scripts/build_utils.sh
-            - ../../build/build
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            # Returns 1 if only .rst and README files were modified
+            echo "Checking if only rst and READMEs were modified"
+            git diff --name-only `git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2` | grep -v '\.rst\|README'
```
Comment (leseb) [inline ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml]: please use `$()`construct instead of backtick.

Comment (djgalloway) [reply ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml]: Done

**Comment by leseb** [comment]
Please rebase on master.

**Comment by djgalloway** [comment]
Done

**Pull request merged**